### ISSUE TITLE
Feature request: read only by default

### ISSUE THREAD

**Issue by cristianarbe** [zim-desktop-wiki/zim-desktop-wiki#623]
We would like to start using Zim at work for a shared wiki between employees but one of the main drawbacks we find is that when you open a notebook the mode is "editing" by default and we would like to avoid accidental deletions without anyone noticing. Would it be possible to have a setting that will toggle whether the default mode is "editing" or "reading"? Or, is there a way we can modify the software so that the default mode is "reading"?

Thanks.

**Comment by jonasstein** [comment]
I suggest to rename it to the more common positive phrase `read only`.
Besides that the  function is inverted like 
`NOT(checkbox selection)`

**Comment by jaap-karssenberg** [comment]
The issue with the editable status button has been fixed in master branch

**Comment by Osndok** [comment]
Just some random thoughts, as I ran into this recently. I found the existing readonly property (that Jaap mentioned) in the `~/.cache/zim/*/state.conf`, but it is a bit hard to administratively maintain (and the user can just toggle the editable flag from the edit menu).

For my case (which is obsolete now), we needed a way for the same data set to be read only in some stations, but writable at others and updatable by a background process... so a flag in the notebook repo (`notebook.zim` file) would not work (as it would be carried around with the rest of the data), and we didn't really want to toggle the directory permissions back-and-forth to allow a background process to update it while keeping zim seeing it read-only.

For something more solid, I suppose we could have a `readonly` property in the `NotebookInfo` list that would have the same effect as if [the notebook directory were readonly](https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/d1560684128d727c0374b26027421a724ba86a78/zim/notebook/notebook.py#L276)... risking, of course, a growing number of ways to check if a new user finds that a notebook is undesirably read-only.
